recipes/re2/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "re2" %}
+{% set version = "2016.11.01" %}
+{% set dashversion = version.replace('.', '-') %}
+{% set sha256 = "01ee949f03e1c4057dc533cf139f967fb1b427015769d53b9ee07757631e9669" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://github.com/google/{{ name }}/archive/{{ dashversion }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not py35]
+  #features:
+  #  - vc9     # [win and py27]
+  #  - vc10    # [win and py34]
+  #  - vc14    # [win and py35]
+  detect_binary_files_with_prefix: True
+  #msvc_compiler: 14.0
+
+requirements:
+  build:
+    - toolchain
+    - cmake >=3.1
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libre2.so  # [linux]
+    - test -f ${PREFIX}/lib/libre2.dylib  # [osx]
+    - test -f ${PREFIX}/include/re2/re2.h  # [unix]
+    - if not exist %PREFIX%\\Library\\include\\re2\\re2.h exit 1  # [win]
+    - if not exist %PREFIX%\\Library\\lib\\re2.lib exit 1  # [win]
```

Comment:
You need the dll because it is a shared build, but it is not getting installed, try the following patch
```diff
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3c90f5d..01bdc72 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -128,4 +128,5 @@ set(RE2_HEADERS
     )
 
 install(FILES ${RE2_HEADERS} DESTINATION include/re2)
-install(TARGETS re2 ARCHIVE DESTINATION lib LIBRARY DESTINATION lib)
+install(TARGETS re2 ARCHIVE DESTINATION lib LIBRARY DESTINATION lib
+                    RUNTIME DESTINATION bin)
```